### ISSUE TITLE
gcc ABI incompatibility

### ISSUE THREAD

**Issue by c0g** [GPflow/GPflow#227]
`setup.py` assumes tensorflow was compiled by gcc 4.x. My tensorflow is compiled with 5.x, so the line:

```
    if gcc_version == 5:
        compile_command += " -D_GLIBCXX_USE_CXX11_ABI=0"
```

causes an undefined symbol error. Removing it makes everything work. I'm not sure if there's a way to work out which ABI tensorflow was compiled with.

Probably not going to get many people, but might worth making a note in the readme or something.

**Comment by icouckuy** [comment]
I think everyone will have this issue if they download the pip package of tf0.11. I have a PR on this but it hasnt been merged it

**Comment by markvdw** [comment]
Is this the PR you're talking about? https://github.com/GPflow/GPflow/pull/222/

**Comment by icouckuy** [comment]
Sorry, I was talking about the official binaries which is different from cog's problem. I 'm  assuming most people would install the official TF binaries on a recent (i.e., gcc6) linux distribution. At least everyone in our group had the same problem of undefined symbols (fixed with #222).

**Comment by alexggmatthews** [comment]
We no longer have user ops.
